frontend/packages/console-shared/src/components/dynamic-form/fields.tsx

```diff
@@ -1,23 +1,27 @@
 import * as _ from 'lodash';
+import * as classnames from 'classnames';
 import * as React from 'react';
-import { FieldProps } from 'react-jsonschema-form';
+import { JSONSchema6 } from 'json-schema';
+import { getUiOptions } from 'react-jsonschema-form/lib/utils';
+import { FieldProps, UiSchema } from 'react-jsonschema-form';
+import SchemaField, {
+  SchemaFieldProps,
+} from 'react-jsonschema-form/lib/components/fields/SchemaField';
 import { LinkifyExternal, SelectorInput, Dropdown } from '@console/internal/components/utils';
+import { AccordionContent, AccordionItem, AccordionToggle, Switch } from '@patternfly/react-core';
+import { MatchExpressions } from '@console/operator-lifecycle-manager/src/components/descriptors/spec/match-expressions';
 import { ResourceRequirements } from '@console/operator-lifecycle-manager/src/components/descriptors/spec/resource-requirements';
-import { FieldSet, FormField } from './templates';
-import { ConfigureUpdateStrategy } from '@console/internal/components/modals/configure-update-strategy-modal';
+import {
+  ConfigureUpdateStrategy,
+  UPDATE_STRATEGY_DESCRIPTION,
+} from '@console/internal/components/modals/configure-update-strategy-modal';
 import {
   NodeAffinity,
   PodAffinity,
 } from '@console/operator-lifecycle-manager/src/components/descriptors/spec/affinity';
-import { MatchExpressions } from '@console/operator-lifecycle-manager/src/components/descriptors/spec/match-expressions';
-import { getUiOptions } from 'react-jsonschema-form/lib/utils';
-import { Switch } from '@patternfly/react-core';
-import SchemaField, {
-  SchemaFieldProps,
-} from 'react-jsonschema-form/lib/components/fields/SchemaField';
-import { getSchemaErrors } from './utils';
+import { getSchemaErrors, useSchemaDescription, useSchemaLabel } from './utils';
 
-export const DescriptionField: React.FC<FieldProps> = ({ id, description }) =>
+const Description = ({ id, description }) =>
```

Comment:
Pulled this out so that it can be used in components that don't have all `FieldProps`.